pkg/madmin/README.md

```diff
@@ -262,6 +262,24 @@ Fetches Mem utilization for all cluster nodes. Returned value is in Bytes.
 | `mem.Usage.Mem`   | _uint64_ | The total number of bytes obtained from the OS         |
 | `mem.Usage.Error` | _string_ | Error (if any) encountered while accesing the CPU info |
 
+<a name="ServerVaultInfo"></a>
+### ServerVaultInfo() (ServerVaultInfo, error)
+
+Fetches Vault information. Returned value is in Bytes.
```

Comment:
What does "Returned value is in Bytes" mean here? Aren't we returning a value of the type `ServerVaultInfo`?

Reply:
Removed the line , but wanted a clarification, The same thing is in `ServerMemUsageInfo` , `ServerCPULoadInfo` , `ServerDrivesPerfInfo` and they all return the value but the docs says `Returned value is in Bytes.` Need your input on this.

Reply:
@sinhaashish It is not clear to me why that line is in the docs. We should fix them in a separate PR.